# SetPos.srv

float64 pos_x
float64 pos_y
float64 pos_z
---
float64 set_x
float64 set_y
float64 set_z
